config/locales/admin/cs.yml

```
@@ -21,6 +24,7 @@ cs:
           only_one_active_label_per_priority: "Only one active label with position %{position} can exist at a time"
           only_one_active_label_per_type: Only one active label of a given type is allowed
           no_overlapping_validity_dates: The validity dates of a given label type cannot overlap
+          valid_color_format: "Must be a valid color format (e.g., #FF0000)"
```

```suggestion
          valid_color_format: 'Must be a valid color format (e.g., #FF0000)'
```
do wszystkich tłumaczeń ta uwaga